test/cmocka/src/math/trig/asin_32b_fixed.c

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2018 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <math.h>
+#include <cmocka.h>
+
+#include <sof/audio/format.h>
+#include <sof/math/trig.h>
+#include <sof/common.h>
+/* 'Error (max = 0.000000027939677), THD+N  = -157.454534077921551 (dBc)' */
+#define CMP_TOLERANCE 0.000000068141916
+#define _M_PI		3.14159265358979323846	/* pi */
+/* Reference table generated by asin(), gcc-4.3.2 */
+static const float asin_ref_table[] = {
```

Comment:
The same for asin, could the 16 and 32 bit tables be the same with common included file.

Reply:
Yes - they are same. Yes they can be included in common header file

Reply:
Degree table as well